README.adoc

```diff
@@ -260,51 +314,59 @@ After you see the following message, your application server in dev mode is read
 ************************************************************************
 *    Liberty is running in dev mode.
 ----
-
 {empty} +
-
 [role="code_command hotspot file=0", subs="quotes"]
 ----
 #Create the SystemBrokerIT class file.#
 `system/src/test/java/it/io/openliberty/guides/system/SystemBrokerIT.java`
 ----
 
-The connection information for the Pact Broker is provided with the [hotspot=connectionInfo file=0]`@PactBroker` annotation. The dependency also provides a JUnit5 Invocation Context Provider with the [hotspot=invocation file=0]`pactVerificationTestTemplate()` method to generate a test for each of the interactions.
+The connection information for the Pact Broker is provided with the [hotspot=connectionInfo file=0]`@PactBroker` annotation.
+The dependency also provides a JUnit5 Invocation Context Provider with the
+[hotspot=invocation file=0]`pactVerificationTestTemplate()` method to generate a test for each of the interactions.
 
-The [hotspot=publish file=0]`pact.verifier.publishResults` property is set to `true` so that the results are sent to the Pact Broker after the tests are completed.
+The [hotspot=publish file=0]`pact.verifier.publishResults` property is set to `true` so
+that the results are sent to the Pact Broker after the tests are completed.
 
 The test target is defined in the [hotspot=context file=0]`PactVerificationContext` context to point to the running endpoint of the `system` microservice.
 
-The [hotspot=state file=0]`@State` annotation must match the [hotspot=given file=2]`given()` parameter that was provided in the `inventory` test class so that Pact can identify which test case to run against which endpoint.
+The [hotspot=state file=0]`@State` annotation must match the [hotspot=given file=2]`given()` parameter
+that was provided in the `inventory` test class so that Pact can identify which test case to run against which endpoint.
 
 [role="code_command hotspot file=1", subs="quotes"]
 ----
 #Replace the system Maven project file.#
 `system/pom.xml`
 ----
 
-The `system` microservice uses the [hotspot=pactDependency file=1]`junit5` pact provider dependency to connect to the Pact Broker and verify the pact file.
-Ideally, in a CI/CD build pipeline, the [hotspot=version file=1]`pact.provider.version` element is dynamically set to the build number so that it's easier to identify at which point a breaking change is introduced.
+The `system` microservice uses the [hotspot=pactDependency file=1]`junit5` pact provider dependency
+to connect to the Pact Broker and verify the pact file.
+Ideally, in a CI/CD build pipeline, the [hotspot=version file=1]`pact.provider.version` element is
+dynamically set to the build number so that it's easier to identify at which point a breaking change is introduced.
```

Comment:
...so that it's easier to identify at which point a breaking change is introduced.

--->

so that you can identify where a breaking change is introduced.